### PR TITLE
core(layout-shift-elements): add CLS savings

### PR DESCRIPTION
**Pull request by adamraine**

### CODE REVIEW

File: core/audits/layout-shift-elements.js [hunk 1]
```diff
@@ -61,6 +64,8 @@ class LayoutShiftElements extends Audit {
         {nodeCount: clsElementData.length});
     }
 
+    details.metricSavings = {CLS: clsSavings};
```
Comment (connorjclark) [inline core/audits/layout-shift-elements.js:67]: silly question but.... won't this just always equal the total CLS score?

Comment (adamraine) [reply core/audits/layout-shift-elements.js:67]: This audit is just the top 5 elements

Comment (brendankenny) [reply core/audits/layout-shift-elements.js:67]: _shouldn't_ it equal the total CLS?

Comment (paulirish) [reply core/audits/layout-shift-elements.js:67]: > don't we want it to equal the total CLS?

IMO yeah

Comment (adamraine) [reply core/audits/layout-shift-elements.js:67]: Depends how you want to look at it...

On one hand, you can't fix this audit without fixing every LS on the page and therefore fix all CLS.

On the other hand, the report we give you can only help with the elements listed, so the potential savings for that report is limited.

Seems like most of us prefer the former interpretation, it's likely users see it that way as well. Also the "top 5" nature of the audit is just kinda hinted at in the description, not the most obvious. I'm fine changing it.

**Review by connorjclark — APPROVED** [review]

**Pull request merged**